crates/sui-core/src/authority_aggregator.rs

```diff
@@ -833,19 +837,56 @@ where
     pub async fn get_latest_system_state_object_for_testing(
         &self,
     ) -> anyhow::Result<SuiSystemState> {
-        let object = self
-            .get_latest_object_version_for_testing(SUI_SYSTEM_STATE_OBJECT_ID)
-            .await?;
-        let system_state_object = bcs::from_bytes::<SuiSystemState>(
-            object
-                .data
-                .try_as_move()
-                .ok_or(UserInputError::MovePackageAsObject {
-                    object_id: object.id(),
-                })?
-                .contents(),
-        )?;
-        Ok(system_state_object)
+        #[derive(Debug, Default)]
```

Comment:
we should delete `get_latest_object_version_for_testing` (btw I don't understand what it wants to do)

Reply:
It's unfortunately currently used by the benchmark code